addons/sale/wizard/sale_make_invoice_advance.py

```diff
@@ -153,11 +185,9 @@ def _check_down_payment_product_is_valid(self):
     #=== ACTION METHODS ===#
 
     def create_invoices(self):
+        self._check_amount_is_positive()
```

Comment:
Why can't it stay as a standard constraint anymore ?

Reply:
Isn't there another way to do it (and avoid saving the wizard if you click on this action) ?
(Maybe special="cancel" but not really sure it works in this case :thinking:)

Reply:
I tried `special="cancel"` but it did not work :confused: 
And I can't think of any other way to prevent the auto-save :sweat_smile: 
I thought checking the amount in the invoice creation was achieving the desired outcome while being less restrictive than the constraint, solving the problem with no side effect that I could think of :slightly_smiling_face:

Reply:
Opening in a modal would fix that, not sure if acceptable